package.json

```diff
@@ -31,6 +31,7 @@
   ],
   "dependencies": {
     "normalize-url": "^1.3.0",
+    "unescape": "^1.0.1",
```

Comment:
No need for a module. You can use `decodeURIComponent`.

Reply:
decodeURIComponent doesnt handle html codes such as &amp; which this library does